### PR TITLE
[Draft] Add Stimulus + Turbolinks post

### PR DESCRIPTION
**Pull request by pascalw**
🌶 Spice up your server-side rendered apps with Stimulus and Turbolinks

### CODE REVIEW

File: drafts/spice-up-your-server-side-rendered-apps-with-stimulus-and-turbolinks.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# 🌶 Spice up your server-side rendered apps with Stimulus and Turbolinks
+
+These days, SPA's (Single Page Applications) are an undeniable trend on the web. However, I personally think there are many use cases where a 'traditional' server-side rendered application still makes sense, for example for CRUD-style / admin applications or in general when your application doesn't require highly interactive features.
```
Comment (rbr) [inline drafts/spice-up-your-server-side-rendered-apps-with-stimulus-and-turbolinks.md]: Would split this in multiple sentences. It feels very long ;)

File: drafts/spice-up-your-server-side-rendered-apps-with-stimulus-and-turbolinks.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+For use cases like these, it's generally more convenient and faster to use server side rendering. Still, also these kinds of applications can benefit from small bits of interactivity like a `copy to clipboard` feature, selecting multiple checkboxes in a table at once etc.
```
Comment (rbr) [inline drafts/spice-up-your-server-side-rendered-apps-with-stimulus-and-turbolinks.md]: `Still, also` feels a bit like a tautology

File: drafts/spice-up-your-server-side-rendered-apps-with-stimulus-and-turbolinks.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+Stimulus also works very well combined with [Turbolinks](https://github.com/turbolinks/turbolinks/). Turbolinks is a small library you can drop into your page which will massively speed up your app, with hardly any work required on your part. It does this by applying a [neat trick](https://github.com/turbolinks/turbolinks/#navigating-with-turbolinks); it fetches subsequent pages in your app via Ajax and merges the new page into the current page. This reduces the work the browsers have to do for each navigation greatly.
+
+Combining Turbolinks and Stimulus will make your app feel very responsive, interactive where it needs to be and best of all you don't have to re-architect your app to achieve this.
+
+Note that Turbolinks originated from Ruby on Rails, but works with any server side rendering technology. At Kabisa we've used Turbolinks together with Elixir Phoenix for example.
+
+If you want to learn more about Stimulus be sure to check out its [handbook](https://stimulusjs.org/handbook/introduction). Refer to the [Turbolinks docs](https://github.com/turbolinks/turbolinks/#attaching-behavior-with-stimulus) for more information about using Turbolinks and Stimilus together.
```
Comment (rbr) [inline drafts/spice-up-your-server-side-rendered-apps-with-stimulus-and-turbolinks.md]: `Stimilus` Typo ;)

File: drafts/spice-up-your-server-side-rendered-apps-with-stimulus-and-turbolinks.md (hunk 2 above)
Comment (Tuxified) [inline drafts/spice-up-your-server-side-rendered-apps-with-stimulus-and-turbolinks.md]: `these kind of applications`, kinds (= plural) would indicate multiple kinds of applications, sounds way more than previously mentioned

File: drafts/spice-up-your-server-side-rendered-apps-with-stimulus-and-turbolinks.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
+# 🌶 Spice up your server-side rendered apps with Stimulus and Turbolinks
+
+These days, SPA's (Single Page Applications) are an undeniable trend on the web. However, I personally think there are many use cases where a 'traditional' server-side rendered application still makes sense, for example for CRUD-style / admin applications or in general when your application doesn't require highly interactive features.
+
+For use cases like these, it's generally more convenient and faster to use server side rendering. Still, also these kinds of applications can benefit from small bits of interactivity like a `copy to clipboard` feature, selecting multiple checkboxes in a table at once etc.
+
+[Stimulus](https://stimulusjs.org) is a relatively new Javascript framework designed specifically to be used with server-side rendered applications. It provides tools to bind Javascript to your server-side rendered HTML. It's modest in feature set, but provides just the right tools, abstractions and conventions to add these little nuggests of interactivity to your application in a sane way.
```
Comment (Tuxified) [inline drafts/spice-up-your-server-side-rendered-apps-with-stimulus-and-turbolinks.md]: `designed specifically to be used with..` sounds like a mouthful, how about: `designed specifically for server-side ..`

File: drafts/spice-up-your-server-side-rendered-apps-with-stimulus-and-turbolinks.md (hunk 3 above)
Comment (Tuxified) [inline drafts/spice-up-your-server-side-rendered-apps-with-stimulus-and-turbolinks.md]: `Refer to the [Turbolinks docs]` sounds imperative, why should I refer the turbolinks docs? Whom should I refer it to? Perhaps something like: `The turbolinks docs contain a section on using Turbolinks and Stimilus together ..`?

File: drafts/spice-up-your-server-side-rendered-apps-with-stimulus-and-turbolinks.md (hunk 1 above)
Comment (ruisalgado) [reply drafts/spice-up-your-server-side-rendered-apps-with-stimulus-and-turbolinks.md]: `. For example for CRUD-style applications, `

File: drafts/spice-up-your-server-side-rendered-apps-with-stimulus-and-turbolinks.md (hunk 3 above)
Comment (pascalw) [reply drafts/spice-up-your-server-side-rendered-apps-with-stimulus-and-turbolinks.md]: Nice catch!

Comment (pascalw) [reply drafts/spice-up-your-server-side-rendered-apps-with-stimulus-and-turbolinks.md]: Good point.

Changed it to:

```
If you want to learn more about Stimulus be sure to check out its handbook. The Turbolinks docs are also excellent and even have a section on using Turbolinks together with Stimulus.
```

**Pull request merged**